stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -489,7 +491,7 @@ class FPDF(GraphicsStateMixin):
         ncols: int = 1,
         gutter: float = 10,
         balance: bool = False,
-        text_align: Align | _AlignLiteral = "LEFT",
+        text_align: str | _TextAlign | tuple[_TextAlign | str, ...] = "LEFT",
```

Comment:
Why allow `str` here? This seems to unnecessarily widen this type to arbitrary strings, which are not allowed at runtime. Also, this makes this type redundant with `str | Align` (as `_AlignLiteral` is a subtype of `str`).

Reply:
this is exact `Literal[]` vs `str | Literal[]` case. I can do whatever is best.